tests/responses/__init__.py

```diff
@@ -40,6 +42,7 @@ def _validate(self) -> None:
                     raise ValueError("You can't test the type availability, if already testing its value/type.")
 
     @abc.abstractmethod
+    @pytest.fixture(scope="class")
     def build(self) -> Any:  # noqa: ANN401
```

Comment:
Just an unrelated question, why do we annotate this as `Any`? Shouldn't it be `BaseStatusResponse`? (we can still make build methods return a more specific types in concrete impls with that, just not a less specific one)